scan-engine/proguard-rules.pro

```diff
@@ -27,8 +27,8 @@
 -keep class de.tillhub.scanengine.ScanEngine$Companion { *; }
 -keep class de.tillhub.scanengine.common.SingletonHolder { *; }
 
--keep class de.tillhub.scanengine.data.ScanEvent { *; }
--keep class de.tillhub.scanengine.data.ScanEvent* { *; }
+-keep class de.tillhub.scanengine.data.ScannerEvent { *; }
+-keep class de.tillhub.scanengine.data.ScannerEvent* { *; }
```

Comment:
You might need more here now. Please test with Pos Go and let me know.

Reply:
this is enough , able to see these on posgo